pyked/chemked.py

```diff
@@ -37,6 +37,8 @@
 Apparatus.institution.__doc__ = '(`str`) The institution where the experiment is located'
 Apparatus.facility.__doc__ = '(`str`) The particular experimental facility at the location'
 
+Composition = namedtuple('Composition', 'species_name InChI SMILES atomic_composition amount')
```

Comment:
This looks fine, make sure you add the documentation as is done for the other namedtuples